src/devices/Ft4222/PinDrivingStrength.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Device.Ft4222
+{
+    /// <summary>
+    /// Intensity for SPI in Milli Amperes
+    /// </summary>
+    internal enum PinDrivingStrength
+    {
+        /// <summary>
+        /// 4 Milli Amperes
+        /// </summary>
+        Intensity4Ma = 0,
```

Comment:
I'd go with regular casing here since this is just confusing (Intensity4mA etc) - I feel like this is one of the places where guidelines makes it confusing